### DISCUSSION TITLE
Oh my Posh icons working fine in terminal but not in VSCode

### DISCUSSION THREAD

**Discussion by AmodeusR** [JanDeDobbeleer/oh-my-posh#4846] [Troubleshoot]
Could someone help me with this problems? Oh My Posh theme works fine in a separated bash Terminal in my computer, but in VSCode there are a bunch of icons missing and I don't know why. It literally happen after I spend some time not using my PC (because it was broken). I needed to swap my CPU and motherboard and now when I come back to it it's like this, not working in VSCode. It worked fine back then. Font is properly configured with `"terminal.integrated.fontFamily": "FiraCode Nerd Font"`. If I mistype the font's name, the font in the VSCode terminal breaks altogether, so it's having an effect, but not completely. I tried reinstalling all the fonts but nothing. What could it be?

Terminal
![image](https://github.com/JanDeDobbeleer/oh-my-posh/assets/29683600/2c84313a-43c8-4260-bc2b-aecf9c1b7854)

VSCode
![image](https://github.com/JanDeDobbeleer/oh-my-posh/assets/29683600/21262d82-24d1-4f9f-9c2c-900b4e608790)

**Comment by JanDeDobbeleer** [comment]
Utf8?

**Reply by AmodeusR** [reply]
I would suppose it is. How can I check?

**Reply by JanDeDobbeleer** [reply]
@AmodeusR you can try forcing the shell to UTF8 and see if the error persists.

**Reply by AmodeusR** [reply]
I tried searching how to do it, but I couldn't find a way. But how could that be the problem, if it works fine when the terminal is opened by itself, but not in VSCode? The problem is in VSCode somehow.

**Reply by JanDeDobbeleer** [reply]
@AmodeusR good question. I'd ask the vscode team. The rendering is always the responsibility of the terminal, also for fonts.
